include/flatbuffers/idl.h

```diff
@@ -167,6 +170,7 @@ struct Type {
   StructDef *struct_def;  // only set if t or element == BASE_TYPE_STRUCT
   EnumDef *enum_def;      // set if t == BASE_TYPE_UNION / BASE_TYPE_UTYPE,
                           // or for an integral type derived from an enum.
+  short fixed_length;     // only set if t == BASE_TYPE_ARRAY
```

Comment:
Why type of `fixed_length` is `signed short`?
Somewhere in the code are negative values used?